src/main/java/training/Roller/SimulationRoller/SimulationRoller.java

```diff
@@ -0,0 +1,44 @@
+package training.Roller.SimulationRoller;
+
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.math.system.plant.DCMotor;
+import edu.wpi.first.wpilibj.simulation.DCMotorSim;
+import training.Roller.IRoller;
+
+public class SimulationRoller implements IRoller {
+
+	private final DCMotorSim motor;
```

Comment:
use SimpleMotorSimulation

Reply:
but this isn't a Falcon500Foc

Reply:
Also I tried to use DutyCycleCount as a controltype and it doesn't seems to move the elbow, unlike setPower.